diagnostic.md

```diff
@@ -49,7 +51,11 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+@function tint ($base-color, $percentage) {
+  @return
+  $percentage = 20%;
+  $base-color = $conflower/$percentage
+}
 ```
```

Comment:
I see what you were going for, but this is too much like javascript and not enough SASS syntax.